docs/source/starter/style_guide.rst

```diff
@@ -24,7 +26,7 @@ In Lightning we differentiate between a system and a model.
 
 A model is something like a resnet18, RNN, etc.
 
-A system defines how a collection of models interact with each other. Examples of this are:
+A system defines how a collection of models interact with each other with user-defined training/evaluation logic. Examples of this are:
 
 * GANs
 * Seq2Seq
```

Comment:
The below bullet could be simply deleted.
```
* etc
```

Reply:
why ? 🤔

Reply:
It just looks unnecessary to me, but if you think it looks nicer as is, let's keep it :)